.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+
+New Feature
+
+- [ ] Clean code (small functions, proper variable names, PEP8 etc)
+- [ ] Doctests
+- [ ] Unit-tests
+- [ ] Documentation
+- [ ] New line item in releases.md
```

Comment:
Should be `[releases.md](releases.md)` to render as: [releases.md](releases.md)

Reply:
nice one, thanks